contrib/cronie/template.py

```diff
@@ -0,0 +1,41 @@
+pkgname = "cronie"
+pkgver = "1.7.1"
+pkgrel = 0
+build_style = "gnu_configure"
+configure_args = [
+    "--enable-anacron",
+    "--with-inotify",
+    "--with-pam",
+    "--without-selinux",
+]
+make_cmd = "gmake"
+hostmakedepends = ["gmake", "autoconf", "automake"]
+makedepends = ["linux-pam-devel", "musl-obstack-devel"]
+depends = ["bash", "debianutils"]
```

Comment:
afaict the bash here is just because the default 0hourly sets SHELL=bash, but could make it sh instead so it doesn't pull in bash just for the default config (and people can still set it if they want it)